projects/angular/src/modal/modal.spec.ts

```diff
@@ -258,6 +259,27 @@ describe('Modal', () => {
     flushAndExpectOpen(fixture, false);
   }));
 
+  it('focus trap remain active after clicking on backdrop', fakeAsync(() => {
+    const backdrop: HTMLElement = compiled.querySelector('div.modal-backdrop');
+    const titleWrapperElement: HTMLElement = compiled.querySelector('div.modal-title-wrapper');
+    const focusStealButton: HTMLElement = compiled.querySelector('button.btn.to-focus');
+
+    fixture.componentInstance.staticBackdrop = true;
+    fixture.detectChanges();
+
+    // Just make sure we have the "x" to close the modal,
+    // because this is different from the clrModalClosable option.
+    expect(compiled.querySelector('.close')).not.toBeNull();
+    expect(titleWrapperElement).toEqual(document.activeElement as HTMLElement);
+
+    focusStealButton.focus();
+    expect(focusStealButton).toEqual(document.activeElement as HTMLElement);
```

Comment:
```suggestion
    expect(document.activeElement ).toEqual(focusStealButton);
```

Assertions should be written as `expect(actualValue).toBe(expected value);`.

Reply:
corrected in commit 31f7c5e